packages/@aws-cdk/aws-codestarnotifications/lib/source.ts

```diff
@@ -0,0 +1,28 @@
+import { INotificationRule } from './notification-rule';
```

Comment:
This file should be called `notification-rule-source.ts`.

Reply:
Already changed.